string object_name
string model_name
---
